# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
int32 type
float64 x_des
float64 y_des
float64 theta_des
